### PR TITLE
Add Cloud Foundry to Reference / Providers nav; add 'alpha' note for CF

### PR DESCRIPTION
**Pull request by fifthposition**

### CODE REVIEW

File: reference/providers/cf.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ sidebar:
 
 If you are not familiar with Cloud Foundry (CF) or any of the terms used below, please consult [Cloud Foundry's reference documentation](https://docs.cloudfoundry.org).
 
+**NOTE:** The Cloud Foundry provider is currently in alpha phase.
```
Comment (dorbin) [inline reference/providers/cf.md]: @lwander 
Do you think this should use the alpha notice format you used in places like [this](https://www.spinnaker.io/guides/user/kubernetes-v2/patch-manifest/)?

Comment (lwander) [reply reference/providers/cf.md]: Oh yes that'd be great.

Comment (dorbin) [reply reference/providers/cf.md]: Cool, @fifthposition can you replace with this:
`{% include alpha version="1.10" %}`

Probably version 1.10, right?

**Comment by fifthposition** [comment]
@dorbin @lwander Done! Many thanks. :)

**Review by dorbin — APPROVED** [review]
Thanks, Ben!

**Pull request merged**